tools/pbstate/src/pbstate.c

```diff
@@ -245,9 +243,11 @@ bool pbstate_is_system_active(pbstate_system_t system)
             return config.enable == PB_STATE_A_ENABLED;
         case PBSTATE_SYSTEM_B:
             return config.enable == PB_STATE_B_ENABLED;
+        case PBSTATE_SYSTEM_NONE:
```

Comment:
This is adding new functionality as part of another unrelated commit (the cmake migraiton), consider splitting in two.

Reply:
fixed